server.go

```diff
@@ -503,7 +506,7 @@ func declareAndBind(inputCh *amqp.Channel, binding HandlerBinding, queueDeclareS
 		queueDeclareSettings.Durable,
 		queueDeclareSettings.DeleteWhenUnused,
 		queueDeclareSettings.Exclusive,
-		queueDeclareSettings.NoWait,
+		false, // no-wait.
```

Comment:
Ok this will be the last one because it seems like there's millions of them (still all the same thought)